wpl_cli/src/utils.rs

```diff
@@ -0,0 +1,56 @@
+use std::fs;
+
+use anyhow::{anyhow, Context, Result};
+use bs58::decode;
+use fs::read_to_string as read_path;
+use serde_json::from_str as parse_json_str;
+use solana_sdk::{pubkey::Pubkey, signature::Keypair};
+use wen_new_standard::ID as WEN_NEW_STANDARD_PROGRAM_ID;
+
+pub const MANAGER_SEED: &[u8] = b"manager";
+pub const GROUP_ACCOUNT_SEED: &[u8] = b"group";
+pub const MEMBER_ACCOUNT_SEED: &[u8] = b"member";
+pub const META_LIST_ACCOUNT_SEED: &[u8] = b"extra-account-metas";
+pub const APPROVE_ACCOUNT_SEED: &[u8] = b"approve-account";
+
+pub fn parse_keypair(keypair_path: &String) -> Result<Keypair> {
```

Comment:
This can be used also for custom grinded mints or parts where we do `Keypair::new()`

Reply:
Yup. Will try to put this feature in the upcoming PR :)